onnxmltools/convert/lightgbm/convert.py

```diff
@@ -35,9 +40,10 @@ def convert(model, name=None, initial_types=None, doc_string='', target_opset=No
     :param custom_shape_calculators: a dictionary for specifying the user customized shape calculator
     :return: An ONNX model (type: ModelProto) which is equivalent to the input lightgbm model
     '''
+    if isinstance(model, lightgbm.Booster):
+        model = WrappedBooster(model)
     if initial_types is None:
-        raise ValueError('Initial types are required. See usage of convert(...) in \
-                           onnxmltools.convert.lightgbm.convert for details')
+        initial_types = default_initial_types(model)
```

Comment:
The documentation of the converter should reflect this change of behaviour.

Reply:
Docstring updated